benchmark/smallbank-loader

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
+/* eslint-disable require-jsdoc */
+
+const {ClientService} = require('../');
+
+(async () => {
+  const [properties, numberOfAccount, numberOfThread] = parseArgv();
+  const clientService = new ClientService(properties);
+  const accountsList = createAccountListForThread(
+      numberOfAccount,
+      numberOfThread,
+  ); // [[0, ...], [1, ...], ...]
+
+  const promises = accountsList.map((accounts) =>
+    createAccounts(accounts, clientService),
+  );
+
+  await Promise.all(promises);
+})();
+
+function usage() {
+  const path = require('path');
+  console.log(
+      'Usage: ' +
+      `${path.basename(process.argv[1])} ` +
+      '[--properties PROPERTIES_FILE, --config PROPERTIES_FILE] ' +
+      '[--num-accounts NUM_ACCOUNTS] ' +
+      '[--num-threads NUM_THREADS] ' +
+      '[-h, --help]',
+  );
```

Comment:
Same here. Should the default value for each parameter be mentioned in the usage ?